src/libs/const.js

```diff
@@ -0,0 +1,67 @@
+const ERROR_HEADING = '[ERROR]';
+
+const ERROR_MESSAGE = {
+  MIN_MONEY: '구입금액: 1000 이상 입력할 수 있습니다.',
+  UNIT_MONEY: '구입금액: 1000 단위로 로또를 구입해야 합니다.',
+  INCLUDE_WINNING_NUMBER: '보너스 번호: 이미 당첨 번호에 포함되어 있습니다.',
+};
+
+const MESSAGE = {
+  REQUEST_MONEY: '구입금액을 입력해 주세요.\n',
+  REQUEST_WINNING_NUMBERS: '\n당첨 번호를 입력해 주세요.\n',
+  REQUEST_BONUS_NUMBER: '\n보너스 번호를 입력해 주세요.\n',
+  WINNING_STATS: '\n당첨 통계\n---',
+};
+
+const MONEY = {
+  UNIT: 1000,
+  MIN: 1000,
+};
+
+const LOTTO = {
+  MIN_NUMBER: 1,
+  MAX_NUMBER: 45,
+  NUMBERS_COUNT: 6,
+};
+
+const WINNING_DETAIL = {
+  FIRST: '6개 일치 (2,000,000,000원)',
+  SECOND: '5개 일치, 보너스 볼 일치 (30,000,000원)',
+  THIRD: '5개 일치 (1,500,000원)',
+  FOURTH: '4개 일치 (50,000원)',
+  FIFTH: '3개 일치 (5,000원)',
+};
+
+const PRIZE = {
+  FIRST: 2000000000,
+  SECOND: 30000000,
+  THIRD: 1500000,
+  FOURTH: 50000,
+  FIFTH: 5000,
+};
```

Comment:
자바스크립트에서는 가독성을 위해 숫자를 표현할 때 언더바 `_`로 자릿수를 구분 지을 수 있다는 것 참고하시면 좋을 것 같습니다👍

```suggestion
const PRIZE = {
  FIRST: 2_000_000_000,
  SECOND: 30_000_000,
  THIRD: 1_500_000,
  FOURTH: 50_000,
  FIFTH: 5_000,
};
```

Reply:
좋은 꿀팁감사합니다!